com.unity.render-pipelines.core/Runtime/Debugging/Prefabs/Resources/DebugUICanvas.prefab

```diff
@@ -127,7 +127,7 @@ MonoBehaviour:
     prefab: {fileID: 224720214277421396, guid: f22bcc84a5f4a1944b075a2c4ac71493, type: 3}
   - type: UnityEngine.Rendering.DebugUI+FloatField, Unity.RenderPipelines.Core.Runtime,
       Version=0.0.0.0, Culture=neutral, PublicKeyToken=null
-    prefab: {fileID: 224720214277421396, guid: 7d4fd3415ea7dd64bbcfe13fb48a730b, type: 3}
+    prefab: {fileID: 224720214277421396, guid: d8c744701b43c864b88e7f8144e19bc5, type: 3}
```

Comment:
Is this intentional change?

Reply:
Yes and no. I think what happened is I made a local copy under Assets, of a prefab that existed under under Packages/RP Core. When I reopened the project, Unity found duplicate GUIDs and assigned a new GUID for the prefab under Packages. I accidentally submitted that change, which lead to some other prefabs having their prefab links broken, so this line is fixing that prefab link.

Do you think I should try and revert back to the original GUIDs?